content/events/2020-montreal/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Montréal
+date: '2020-09-15'
+location:
+  countrycode: ca
+  country: Canada
+  city: Montréal
+  url: https://www.google.com/maps/place/%C3%89cole+de+technologie+sup%C3%A9rieure+%C3%89TS/@45.4945497,-73.5644347,17z/data=!3m1!4b1!4m5!3m4!1s0x4cc91a60aa839707:0xb732a719a45c45f6!8m2!3d45.494546!4d-73.562246
+  venue: ETS
+map: ets.png
+sched_slug: kubernetescommunitydaymontreal
+social:
+  twitter: 'https://twitter.com/kcd_montreal'
+sponsors:
+#- level: platine
+#  size: large
+#  orgs:
+#  - name: ???
+#    logo: sponsors/sponsor.svg
+#    url: https://kubernetes.io
+#- level: or
+#  size: medium
+#  orgs:
+#  - name: ???
+#    logo: sponsors/sponsor.svg
+#    url: https://kubernetes.io
+#- level: argent
+#  size: small
+#  orgs:
+#  - name: ???
+#    logo: sponsors/sponsor.svg
+#    url: https://kubernetes.io
+#  - name: ???
+#    logo: sponsors/sponsor.svg
+#    url: https://kubernetes.io
+#- level: bronze
+#  size: small
+#  orgs:
+#  - name: ???
+#    logo: sponsors/sponsor.svg
+#    url: https://kubernetes.io
+#  - name: ???
+#    logo: sponsors/sponsor.svg
+#    url: https://kubernetes.io
+speakers:
+#- name: Phippy
+#  image: speakers/phippy-color.png
+#  twitter: https://twitter.com/cloudnativefdn
+#  bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#  homepage: https://phippy.io
+organizers:
```

Comment:
Julia Simon
https://twitter.com/JuliaSimon14
https://github.com/juliasimon
https://www.linkedin.com/in/jsimon14/
![Julia_pic](https://user-images.githubusercontent.com/17503776/76340076-5ca18e80-62d1-11ea-8098-61b29330fded.png)

Reply:
added